requirements.txt

```diff
@@ -15,4 +15,12 @@ pypdf
 fpdf
 youtube-transcript-api
 pytube
-python-dotenv
\ No newline at end of file
+python-dotenv
+pymupdf
+chromadb
+pdfplumber
+pydantic==1.10.7
```

Comment:
Don't limit the version of the dependencies because it's a good practice to keep them up-to-date to prevent security issues or incompatibilites with other dependencies.